### PR TITLE
Add my solution

### PR DESCRIPTION
**Pull request by vollantre**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

File: ruby_programming/basic_ruby/project_building_blocks.md [hunk 1]
```diff
@@ -1170,4 +1173,4 @@ Next, make sure your method can handle multiple words:
 * [Ifeanyichukwu Amajuoyi's solution](https://github.com/Anyitechs/The_odin_project)
 * [Ashley Cook's Solution](https://github.com/Cooky90/SubStrings/blob/master/Substrings.rb)
```
Comment (ApondoErnest) [inline ruby_programming/basic_ruby/project_building_blocks.md]: [Ernest Apondo's solution] (https://github.com/ApondoErnest/Rock-Paper-Scissors)

**Comment by ryanford-frontend** [comment]
Fix your submission to use the markdown format. I've already asked you twice in your previous submissions.

**Comment by ryanford-frontend** [comment]
https://github.com/TheOdinProject/curriculum/pull/9940#issuecomment-473615405

https://github.com/TheOdinProject/curriculum/pull/9905#issuecomment-472689823

https://github.com/TheOdinProject/curriculum/pull/9820#issuecomment-469954196

File: ruby_programming/basic_ruby/project_building_blocks.md (hunk 1 above)
Comment (ApondoErnest) [inline ruby_programming/basic_ruby/project_building_blocks.md]: [Ernest Apondo's solution](https://github.com/ApondoErnest/Rock-Paper-Scissors)

Comment (leila-alderman) [reply ruby_programming/basic_ruby/project_building_blocks.md]: @ApondoErnest This isn't the correct method for adding your solutions. Please watch this video and follow the instructions there for creating a new pull request to add your solution: https://www.youtube.com/watch?v=V74l_zS1x8E

**Pull request closed without merging**